deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -3,6 +3,11 @@ model_list:
     litellm_params:
       model: text-embedding-3-small
       api_key: "os.environ/OPENAI_API_KEY"
+  - model_name: local-embeddings
+    litellm_params:
+      model: openai/custom-embedding
+      api_base: "os.environ/EMBEDDINGS_ENDPOINT"
+      api_key: "os.environ/EMBEDDINGS_API_KEY"
```

Comment:
I think the idea with this config file was to switch the models we use for each task, so /embeddings /detect-language / safety, etc.

As in, this is the file where we define the models. The env vars are there just for extra flexibility.

So we should just replace the current `model_name: embeddings` to have this config, not create a new model.

(We can switch the aws deployment config to this as well once we 1. host it 2. test it, and are happy with the performance)

Reply:
Isn't the point of the proxy to have many models configured so we only have to change the name of the model we want? Because I think your approach actually ends up being less flexible.

Reply:
Hmm maybe you're right. 

I thought that the whole point of the config was to define 1 `model_name` per prediction task (detect-language, safety, paraphrase, ...). And, each AAQ instance (for use case A, B, C, ..) would have their own proxy server with their own configs. 

But maybe the idea was to (have the option to) serve only 1 proxy server with many different model options for many AAQ instances, and each AAQ instance chooses which model they use by referring to the correct model_name as defined in the config?

cc: @amiraliemami

Reply:
Can we call it huggingface_embeddings? It does not have to be local, it can be on cloud, but just that we can serve opensource embeddings, specifically hf it seems

Reply:
The idea is to have our app code be stable (i.e. always call "openai/embeddings") and we just change the proxy's forwarding to a different model if we want. So yeah, should leave our AAQ code same as before and just change the routing API url and key to be smt else.

Reply:
I will do as we discussed on slack